docker-environment-driver/src/test/java/org/jboss/pnc/environment/docker/DockerEnvironmentDriverRemoteTest.java

```diff
@@ -31,22 +15,36 @@
 import org.jboss.pnc.spi.repositorymanager.RepositoryManagerResult;
 import org.jboss.pnc.spi.repositorymanager.model.RepositoryConnectionInfo;
 import org.jboss.pnc.spi.repositorymanager.model.RepositorySession;
+import org.jboss.pnc.test.category.ContainerTest;
+import org.jboss.pnc.test.category.RemoteTest;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.asset.EmptyAsset;
-import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.jboss.shrinkwrap.resolver.api.maven.Maven;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 
+import javax.inject.Inject;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.Socket;
+import java.util.Map;
+import java.util.UUID;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.*;
+
 /**
  * Unit tests for DockerEnnvironmentDriver
- * 
- * @author Jakub Bartecek <jbartece@redhat.com>
  *
+ * @author Jakub Bartecek <jbartece@redhat.com>
  */
 @RunWith(Arquillian.class)
+@Category({RemoteTest.class, ContainerTest.class})
```

Comment:
In that case I should have used only RemoteTests category. I'll 
introduce this change shortly.

On 03/30/2015 12:51 PM, Marek Novotný wrote:
>
> In 
> docker-environment-driver/src/test/java/org/jboss/pnc/environment/docker/DockerEnvironmentDriverRemoteTest.java 
> <https://github.com/project-ncl/pnc/pull/243#discussion_r27380637>:
>
> >   */
> >  @RunWith(Arquillian.class)
> > +@Category({RemoteTest.class, ContainerTest.class})
>
> I think this test is RemoteTest and is launched in 
> arquillian-weld-se-embedded container, so there will be duplicated 
> container test IMO. WDYT?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/project-ncl/pnc/pull/243/files#r27380637>.
>